### DISCUSSION TITLE
How I can recover my github account

### DISCUSSION THREAD

**Discussion by qantyt** [community/community#54862] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have mistakenly deleted my account. How I can recover it?

**Comment by iamreiyn** [comment]
Hi @qantyt

If you have questions specific to your account like this one, please create a support ticket on the [GitHub Support page](https://support.github.com/request). The support team will be able to assist you with your account-specific issue, thank you.

**Comment by dages00kzbadfbadf** [comment]
hello, my previous account named 'jahonwis' cannot be opened, I can't open it now, what can I do, please help me
